modules/combined/tests/contact_verification/patch_tests/cyl_3/tests

```diff
@@ -25,20 +31,27 @@
     type = 'CSVDiff'
     input = 'cyl3_template1.i'
     csvdiff = 'cyl3_frictionless_kin_check.csv cyl3_frictionless_kin_check_x_disp_0001.csv'
-    cli_args = 'Contact/leftright/model=frictionless Contact/leftright/formulation=kinematic Outputs/file_base=cyl3_frictionless_kin_out Outputs/chkfile/file_base=cyl3_frictionless_kin_check'
+    cli_args = 'Contact/leftright/model=frictionless Contact/leftright/formulation=kinematic
+                Outputs/file_base=cyl3_frictionless_kin_out Outputs/chkfile/file_base=cyl3_frictionless_kin_check
+                Executioner/petsc_options_iname="-pc_type -pc_factor_mat_solver_package"
+                Executioner/petsc_options_value="lu superlu_dist"'
     rel_err = 2e-5
     abs_zero = 1e-4
     max_parallel = 1
     superlu = true
     prereq = 'frictionless_kin_sm'
-    # This test is fails reliably on INTEL compilers with PETSc 3.7.4 too.
-    skip = 'Reevaluate under PETSc 3.7.4. #8200'
+    # This test is fails reliably on INTEL compilers with PETSc 3.7.4 and 3.7.5, so restrict the test to not INTEL
+    compiler = 'GCC CLANG'
```

Comment:
Because 3.7.5 may or may not have the fix, it seems like we can't really reliably do anything based on whether the version is >= 3.7.5.  Maybe we just skip these tests until 3.7.6 comes out?